src/test/scala/is/hail/vds/JoinSuite.scala

```diff
@@ -83,7 +83,7 @@ class JoinSuite extends SparkSuite {
     assert(jLeft.count() == jLeftOrdRDD1.count())
     assert(jLeft.rdd.forall(rv => rv != null))
     assert(jLeft.map { rv =>
-      val ur = new UnsafeRow(localRowType, rv)
+      val ur = new UnsafeRow(localRowType, rv.copy())
```

Comment:
In the tests, just `SafeRow`?  RegionValue.copy() calls the Region constructor which is going to be a problem.